src/core/cdrom.cc

```diff
@@ -1642,13 +1652,18 @@ void PCSX::CDRom::check() {
                 m_cdromId += filename.substr(9, 2);
             }
 
+            if (filename.size() > 0) {
+                m_bootable = true;
+            }
+
             break;
         }
     } else {
         IO<File> psxexe(reader.open("PSX.EXE;1"));
         if (!psxexe->failed()) {
             m_cdromId = "SLUS99999";
             exename = "PSX.EXE;1";
+            m_bootable = true;
```

Comment:
Maybe the `check` function should display a warning at the end that the disc isn't bootable in case this boolean is still false.